docs/data/data-grid/localization/data.json

```
@@ -5,238 +5,238 @@
     "localeName": "Arabic (Sudan)",
     "missingKeysCount": 15,
     "totalKeysCount": 94,
-    "githubLink": "https://github.com/mui/mui-x/blob/next/packages/grid/x-data-grid/src/locales/arSD.ts/"
+    "githubLink": "https://github.com/mui/mui-x/blob/master/packages/grid/x-data-grid/src/locales/arSD.ts/"
```

We can use `-` instead of the branch name which would redirect to the default branch - see https://github.com/mui/mui-x/pull/7968#discussion_r1116902191

```suggestion
    "githubLink": "https://github.com/mui/mui-x/blob/-/packages/grid/x-data-grid/src/locales/arSD.ts/"
```

---

That's a really nice feature of GitHub! 👍 👏